### ISSUE TITLE
[GSv4] Post merge shaping review

### ISSUE THREAD

**Issue by chrissimpkins** [googlefonts/googlesans#243]
Shaping review of the GSv4 source file merge as of https://github.com/googlefonts/googlesans/commit/5189d329cfa3173a207dd6ace356b716d7d9f67d

## No regressions
- calt: includes fix of logo forms after rvrn transition
- case
- ccmp_hebrew
- dlig
- dlig_hebr_IWR 
- dnom 
- jalt_hebr_IWR
- kern_hebr
- liga:  includes fix of logo forms after rvrn transition
- locl
- numr 
- ordn 
- ss01
- ss02
- ss03
- ss04
- ss05
- subs
- sups

## No regressions, JSON shaping defs need update
- pnum: has .BRACKET.18 glyph names that have been removed (done)
- tnum:  has .BRACKET.18 glyph names that have been removed (done)
- smcp_c2sc: has .BRACKET.18 glyph names that have been removed (done)
 
## Regressions

- very minor latn_ccmp and latn_ccmp_smcp regression in Roman 400, 500 weights only

```
hb-shape --variations=wght=400,opsz=18,GRAD=0 GoogleSans\[GRAD,opsz,wght\].ttf i̊i̮̊j̮̊
[dotlessi=0+226|uni030A=0@-313,0+0|dotlessi=2+226|uni032E=2@-312,0+0|uni030A=2@-313,0+0|uni0237=5+226|uni032E=5@-313,-226+0|uni030A=5@-314,0+0]
```

## TODO

- ss06

**Comment by chrissimpkins** [comment]
pnum json definitions updated in 0bfb28c6991b2ce227345f3655c355ba8c5f397b

tnum json definitions updated in ed39f2597ce80519e96f58a1a91c4854b8542fb9

**Comment by madig** [comment]
I think the ccmp regression is just dotlessi advancing 1 font unit less.

**Comment by chrissimpkins** [comment]
> I think the ccmp regression is just dotlessi advancing 1 font unit less.

Seems to be the case.  I looked at the string in Font Goggles.  It looks just as uninterpretable as it did before we merged.  I think that we are good.  😄 

Kidding aside.  It looks fine.  I don't know why it changed.  But it seems like an acceptable regression that we do not need to address.

**Comment by madig** [comment]
I assume it's because the right sidebearing was changed by 1 fU 🤷‍♀️

**Comment by chrissimpkins** [comment]
The smcp_c2sc shaping looks good on visual review.  I'll push an update to our json shaping definition file.  This will eliminate the .BRACKET.18 glyph definitions after our transition away from rvrn for smcp and c2sc glyphs involved

I got a good look at mark positioning too.  Marks appear to be well aligned across Roman and Italic builds.

**Comment by madig** [comment]
What about the de-rvrn'd G issue you flagged somewhere?

**Comment by chrissimpkins** [comment]
> What about the de-rvrn'd G issue you flagged somewhere?

`G` is looking good across opsz axis, including in c2sc forms. 👍

**Comment by chrissimpkins** [comment]
Regression fixed.

JSON updated.

Everything is complete here and will be merged in #240. #240 closes this issue.
